irods/test/pam_interactive_test.py

```diff
@@ -48,9 +48,8 @@ def test_pam_interactive_auth_file_creation(self):
 
         with patch("getpass.getpass", return_value=self.password) as mock_getpass:
             self.sess = make_session(test_server_version=False, env_file=self.env_file_path, authentication_scheme= "pam_interactive")
-            self.assertTrue(self.sess.server_version)
-            self.assertEqual(self.sess.username, self.user)
-            self.assertEqual(self.sess.zone, self.zone)
+            home = self.sess.collections.get(f"/{self.sess.zone}/home/{self.sess.username}")
+            self.assertEqual(home.name, self.sess.username)
```

Comment:
Let's put that comment here as well:
```suggestion
            # Creating a session does not trigger auth, so the home collection is accessed to trigger and confirm auth succeeded
            home = self.sess.collections.get(f"/{self.sess.zone}/home/{self.sess.username}")
            self.assertEqual(home.name, self.sess.username)
```